addons/website/static/src/js/backend/res_config_settings.js

```diff
@@ -54,13 +54,13 @@ const WebsiteCookiesbarField = FieldBoolean.extend({
             title: _t("Please confirm"),
             $content: QWeb.render('website.res_config_settings.cookies_modal_main'),
             buttons: [{
-                text: 'Do not activate',
+                text: _t('Do not activate'),
```

Comment:
Could've used double quote while doing the change but it's ok, keep it like that
Thanks for the PR

Reply:
Yeah I tried to keep the difference as minimal as possible 😄